src/test/java/ToRemoveTest.java

```diff
@@ -17,38 +17,62 @@ public void displayingTestDescription(Method testMethod){
 
     @Test(description = "verify that string is in UpperCase", priority = LOW)
     public void testTestToUpperCase() {
-        String testString = "test";
+        final String testString = "test";
         String actualResult = toUpperCase(testString);
-        String expectedResult = "TEST";
+        final String expectedResult = "TEST";
         final String message = "The text of the line feed to uppercase failed: ";
 
         assertEquals(actualResult, expectedResult, message);
     }
 
+    @DataProvider(name = "multiplyData")
+    public Object[][] dataForMultiply() {
+        return new Object[][]{
+                {2, 3_000_000_000L, 6_000_000_000D},
```

Comment:
Just a note - it would be much more readable and maintainable if you add a comment explaining meaning of each data piece i.e.
// Data-case structure: {the 1-st input value for multiplication | the 2-nd input value | expected multiplication result(the product)}